app/lib/patient_merger.rb

```diff
@@ -115,4 +117,12 @@ def self.call(*args, **kwargs)
   private
 
   attr_reader :patient_to_keep, :patient_to_destroy
+
+  def refresh_statuses
+    Patient
+      .includes(consents: :parent)
```

Comment:
I think we can remove `includes(consents: :parent)`. We're only fetching a single patient record, so the benefit of eager loading appears to be unnecessary, unless I'm missing a trick somewhere.

Reply:
I agree in this case, but the `refresh!` method is designed to work with multiple preloaded patients via the `latest_consents` method, and if we don't preload the `consents` then a strict loading exception is raised.

Reply:
I've updated this PR and introduced a `StatusUpdater` class that handles the calling of `refresh!` which I think helps to clarify the usage.